DOC: exmouth gulf (#74)
jbathmann/pyMANGA: web/content/de/docs/Beispielmodell:_Exmouth_Gulf/Ergebnisse.md

---
title: "Das Modell (Arbeitstitel)"
linkTitle: "Das Modell (Arbeitstitel)"
weight: 3
description:
---

<head>
<style type="text/css">
<!--
#vis {
  border: 1px solid black;
}
#Rahmen {
        border-width: 0.1em; 
        border-style: solid;
        text-align:right;
}
-->
</style>
</head>

# Modellgebiet

Im Modell wird ein Transekt von 185 m Länge und 10 m Breite abgebildet (siehe <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_1">Abbildung 1</a>).
Zur Evaluierung der Modellergebnisse werden sowohl Allometrydaten aus einer Transektenmessung, als auch Messergebnisse einer Langzeitstudie verwendet. In der Langzeitstudie werden die Auswirkungen von Düngung auf das Wachstum von insgesamt 72 Mangroven, von denen jeweils die Hälfte an der land- und an der seeseitigen Seite des Transekts steht, dokumentiert (siehe auch <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_4">Abbildung 4</a>).

<figure>
<a name="Abbildung_1"></a>
<img src="/pictures/exmouth_gulf/Transect_Sketch.png">

> <figcaption><font size = "1"><i><b>Abbildung 1:</b> Modellgebiet</i></font></figcaption>

Reviewer: caption aus paper übernehmen, wenn veröffentlicht

<iframe src="https://www.google.com/maps/d/embed?mid=1EiX5yyZGJgVSu7pueUi5_jK160ndg0tG" width="640" height="480"></iframe>

<br>
<br>
<br>

Die Höhenprofile aus dem digitalen Oberflächenmodell der fünf Transekte (<a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_2"> Abbildung 2 A</a>) wurden übereinander gelegt und aus den fünf Signalen (<a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_2">Abbildung 2 B</a>) die Geländeoberkante mit Hilfe eines digitalen Geländemodells (abgebildet/interpoliert/extrahiert MARKER) (<a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_2">Abbildung 2 C</a>)

<figure>
<a name="Abbildung_2"></a>
<img src="/pictures/exmouth_gulf/dem.png" style="width:75%">
<figcaption><font size = "1"><i><b>Abbildung 2:</b> MARKER</i></font></figcaption>
</figure><br>

MARKER
Soll diese Abbildung hier auch mit rein, oder wird das dann zu viel?

<figure>
<a name="Abbildung_2"></a>
<img src="/pictures/exmouth_gulf/dgm.png" style="width:75%">
<figcaption><font size = "1"><i><b>Abbildung 2:</b> MARKER</i></font></figcaption>
</figure><br>

> MARKER

Reviewer: ist glaub ich zu viel

# Modellierung (Arbeitstitel)

## Modellvarianten

Das Mangrovenwachstum wurde mit Hilfe von drei verschiedenen Modellen simuliert (siehe auch <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Tabelle_1">Tabelle 1</a>). 

> Im Modell <b>"Model Without Feedback"</b> werden die dynamischen Veränderungen der abiotischen Einflüsse (Gezeiten, Grundwasserneubildung und Salzgehalt des Meerwassers (!!!!!!nicht dynamisch abgedildet, oder?) über entsprechende Randbedinungen berücksichtigt. Der Einfluss der Pflanzenwasserentnahme auf den Salzgehalts des Porenwassers wurde nicht abgebildet.

Reviewer: richtig

</figure><p>

Die Mangroven entnehmen dem Untergrund Bodenwasser aus einer Tiefe von 40 cm bis 80 cm unter der Geländeoberkante. <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_3">Abbildung 3</a> zeigt das Modellgebiet (Grau) und den Bereich der Wasserentnahme durch die Mangroven (Blau). Zu beachten ist die 50-fache Skalierung in z-Richtung. 

<figure>
<a name="Abbildung_3"></a>
<img src="/pictures/exmouth_gulf/model_area_legend.png">
<figcaption><font size = "1"><i><b>Abbildung 3:</b> Bereich der Wasserentnahme durch Mangroven</i></font></figcaption>
</figure><p>

Zeitlich diskretisiert wird das Grundwassermodell mit einer Zeitschrittlänge von einer Stunde. Der Tidenhub als dynamische Randbedingung wird mit der Zeitreihe der Jahre 1991 bis 1993, die über die gesamte Modelllaufzeit immer wieder wiederholt wird, abgebildet.

### Baumwachstumsmodell

Da jede Mangrove als einzelnes Individuum abgebildet wird findet eine räumliche Diskretisierung im eigentlichen Sinne nicht statt. Zeitlich wird das Baumwachstumsmodell mit einer Zeitschrittlänge von einem halben Jahr (1 a = 365.25 d) diskretisiert.

## Randbedingungen

### Grundwassermodell

> 

Reviewer: hier findest du im paper eine beschreibung

<td style="text-align: left;">Wurzelradius</td>
<td style="text-align: left;">0.26 m</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline"><em>r</em><sub><em>C</em></sub></span></td>
<td style="text-align: left;">Kronenradius</td>
<td style="text-align: left;">0.2 m</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline"><em>r</em><sub><em>S</em></sub></span></td>
<td style="text-align: left;">Stammradius</td>
<td style="text-align: left;">0.005 m</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline"><em>h</em><sub><em>R</em></sub></span></td>
<td style="text-align: left;">Wurzeltiefe</td>
<td style="text-align: left;">0.004 m</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline"><em>h</em><sub><em>C</em></sub></span></td>
<td style="text-align: left;">Kronenhöhe</td>
<td style="text-align: left;">0.004 m</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline"><em>h</em><sub><em>S</em></sub></span></td>
<td style="text-align: left;">Stammhöhe</td>
<td style="text-align: left;">0.015 m</td>
</tr>
</tbody>
</table>

> 

Reviewer: das braucht am ende alles nochmal nen update

<td style="text-align: left;">Root radius</td>
<td style="text-align: left;">0.26 m</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline"><em>r</em><sub><em>C</em></sub></span></td>
<td style="text-align: left;">Crown radius</td>
<td style="text-align: left;">0.2 m</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline"><em>r</em><sub><em>S</em></sub></span></td>
<td style="text-align: left;">Stem radius</td>
<td style="text-align: left;">0.005 m</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline"><em>h</em><sub><em>R</em></sub></span></td>
<td style="text-align: left;">Root depth</td>
<td style="text-align: left;">0.004 m</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline"><em>h</em><sub><em>C</em></sub></span></td>
<td style="text-align: left;">Crown depth</td>
<td style="text-align: left;">0.004 m</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline"><em>h</em><sub><em>S</em></sub></span></td>
<td style="text-align: left;">Stem heigth</td>
<td style="text-align: left;">0.015 m</td>
</tr>
</tbody>
</table>

-->

> # Runtime [Arbeitstitel]

Reviewer: Ressourcenkonkurrenz?

<figure id="vis">
<a name="Visualisierung_1"></a>
<form oninput="x.value=parseInt(a.value)" id="slider" >
<script type="text/javascript">
 /*<![CDATA[*/
  document.getElementById("slider").addEventListener("input", aktualisiere);
   function aktualisiere() {
	  var TS = (document.querySelector("output[name=x]")) ;
	  var a = '/pictures/exmouth_gulf/TS/ts_'+TS.value+'.png' ;
          document.getElementById("abb").setAttribute('src', a) ;
}
/*]]>*/
</script>
<img src='/pictures/exmouth_gulf/TS/ts_0.png' id="abb">
</br>
</br>
<p align="left">
<font size = "6">&nbsp;  Zeitschritt:&nbsp;&nbsp;&nbsp;&nbsp; </font>
  <input type="range" id="a" min="0" max="1650" step="50"> &nbsp;
<font size = "6">  <output name="x" for="a">0</output> </font>&nbsp;&nbsp;
</p>
</figure>
<figcaption><font size = "1"><i><b>Visualisierung 1:</b> Dynamische Entwicklung der Mangrovenpopulation über die Modellierungszeit</i></font></figcaption>

> <p>

Reviewer: irgendwie nicht interaktiv. :(

Reviewer: Nice! Er geht wieder?!

Reviewer: Vielleicht kannst du die %-Achse auf +-10% setzen? Dann sieht es nach weniger fluktuationen aus. :P

<br>
<br>
<br>

Die Ergebnisse des "<b>Full Models</b>" stimmen mit den gemessenen Felddaten qualitativ überein (siehe <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_4">Abbildung 4</a>). Dies trifft sowohl auf das Baumhöhenprofil (siehe <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_4">Abbildung 4 A</a>) als auch auf das Profil des Salzgehalts des Porenwassers (siehe <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_4">Abbildung 4 B</a>) in dem untersuchten Transekt zu. Insbesondere die Variation des Porenwassersalzgehalts konnte gut abgebildet werden (<a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_4">Abbildung 4 A</a>). Ein Vergleich der Ergebnisse des "Full Models" mit den Ergebnissen der beiden Modellvarianten "Model Without Feedback" und "Model Without Tide" zeigt eine deutlich schlechtere Wiedergabe der gemessenen Felddaten durch die beiden einfacheren Modelle (siehe <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_4">Abbildung 4 C und 4 D</a>). 

<figure>
<a name="Abbildung_4"></a>
<img src="/pictures/exmouth_gulf/Figure_3.png">

> <figcaption><font size = "1"><i><b>Abbildung 4:</b> Ergebnisse</i></font></figcaption>

Reviewer: bissle ausführlichere caption. evtl einfach aus paper. (so wie bei den anderen entnommenen abbildungen)

Author: done

<figure>
<a name="Abbildung_5"></a>
<img src="/pictures/exmouth_gulf/Figure_3_2.png">
<figcaption><font size = "1"><i><b>Abbildung 5:</b> Relative Auswirkung der Nichtberücksichtigung des Tidenhubs ("Model Wihtout Tide") und der Pflanzenwasserentnahme ("Model Without Feedback")</i></font></figcaption>
</figure><p>

Aufgrund der größeren Auswirkungen des Tidenhubs im seenahen Bereich kann das Modell "<b>Without Tide</b>" sowohl die Bäumhöhen als auch den Porenwassersalzgehalt hier nur mit reltiv großer Abweichung im Vergleich zum "<b>Full Model</b>" abbilden. Je weiter man sich aber in Richtung Festland bewergt, desto geringer werden die Wasserstandsschwankungen aufgrund der Gezeiten. Die Baumhöhen und Salzgehalte können in diesem Bereich (x > 75 m) mit geringeren relativen Abweichungen zum Full Model abgebildet werden.

Das Modell "<b>Without Feedback</b>" hat insbesondere im mittleren bis landseitigem Bereich (60 m < x < 165 m) des Transsekts Probleme, die Wachstumshöhe der Mangroven so abzubilden, wie es das "<b>Full Model</b>" macht. 

## Speziendominanz MARKER

> Im vorangegangenem Abschnitt wurde gezeigt, dass pyMANGA mit der Berücksichtigung von Salzkonzentration im Bodenwasser und des Tidenhubs in der Lage ist, die für Mangrovenwälder typischen Waldstrukturen abzubilden. Mit Hilfe der umfangreichen Parametrisierungsmöglichkeiten des Baumwachstumsmodells (siehe hierzu auch MARKER) kann mit pyMANGA auch das Wachstum einzelner bestimmter Individuenarten untersucht werden. Verschiedene Mangrovenarten zum Beispiel weisen verschiedene Toleranzen gegenüber zu hohen Salzgehalten auf.

Reviewer: Gute einleitung. Kannst du die Aussagen aus dem Paper übernehmen?

---
title: "Das Modell"
linkTitle: "Das Modell"
weight: 3
description:
---

<head>
<style type="text/css">
<!--
#vis {
  border: 1px solid black;
}
#Rahmen {
        border-width: 0.1em; 
        border-style: solid;
        text-align:right;
}
-->
</style>
</head>

# Modellgebiet

Im Modell wird ein Transekt von 185 m Länge und 10 m Breite abgebildet (siehe <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_1">Abbildung 1</a>).

> Zur Evaluierung der Modellergebnisse werden sowohl Allometrydaten aus einer Transektenmessung, als auch Messergebnisse einer Langzeitstudie verwendet. In der Langzeitstudie werden die Auswirkungen von Düngung auf das Wachstum von insgesamt 72 Mangroven, von denen jeweils die Hälfte an der land- und an der seeseitigen Seite des Transekts steht, dokumentiert (siehe auch <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_4">Abbildung 4</a>).

Reviewer: Für weitere Arbeiten: Netlify schein latex-format zu verwenden. Das bedeutet, dass ein einfacher Zeilenumbruch nicht als umbruch abgebildet wird. Daher kann jeder Satz eine eigene Zeile bekommen, was das ganze leichter für Versionskontrolle verwendbar macht.
Ein doppelter Zeilen Umbruch in der Source führt zu einem einfachen Umbruch im Target.

Author: Das würde ich bei den Aufräumarbeiten dann vielleicht für die gesamte Homepage ändern - das wäre glaube ich wirklich sehr hilfreich

<figure>
<a name="Abbildung_1"></a>
<img src="/pictures/exmouth_gulf/Transect_Sketch.png">
<figcaption><font size = "1"><i><b>Abbildung 1:</b> Vereinfachte Darstellung der Standortbedingungen und des konzeptionellen Modells</i></font></figcaption>
</figure><br>

Das Modell bildet das Mangrovenwachstum an der in nachfolgender Darstellung zu sehenden Stelle im Exmouth-Golf dar.

<iframe src="https://www.google.com/maps/d/embed?mid=1EiX5yyZGJgVSu7pueUi5_jK160ndg0tG" width="640" height="480"></iframe>

<br>
<br>
<br>

> Die Höhenprofile aus dem digitalen Oberflächenmodell der fünf Transekte (<a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_2"> Abbildung 2 A</a>) wurden übereinander gelegt und aus den fünf Signalen (<a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_2">Abbildung 2 B</a>) die Geländeoberkante mit Hilfe eines digitalen Geländemodells (abgebildet/interpoliert/extrahiert MARKER) (<a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_2">Abbildung 2 C</a>)

Reviewer: ```suggestion
Die Höhenprofile aus dem digitalen Oberflächenmodell der fünf Transekte (<a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_2"> Abbildung 2 A</a>) wurden übereinander gelegt und aus den fünf Signalen (<a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_2">Abbildung 2 B</a>) die Geländeoberkante aus der Grundlinie der Signals interpoliert (<a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_2">Abbildung 2 C</a>)
```

<figure>
<a name="Abbildung_2"></a>
<img src="/pictures/exmouth_gulf/dem.png" style="width:75%">
<figcaption><font size = "1"><i><b>Abbildung 2:</b> MARKER</i></font></figcaption>
</figure><br>

# Modellierung (Arbeitstitel)

> ## Modellvarianten

Reviewer: Da ist irgendwas mit dem Layout schiefgegangen

    <font color="green" size="5"> <b> &#10004; </b> </font>
  </td>
 </tr>
 <tr>
  <td width="27%" style="text-align: center; vertical-align: middle;">
   Model Without Tide
  </td>
  <td width="23%" style="text-align: center; vertical-align: middle;">
    <font color="red" size="5"> <b> &#10008; </b> </font>
  </td>
  <td width="26%" style="text-align: center; vertical-align: middle;">
    <font color="green" size="5"> <b> &#10004; </b> </font>
  </td>
  <td width="23%" style="text-align: center; vertical-align: middle;">
    <font color="green" size="5"> <b> &#10004; </b> </font>
 </tr>
 <tr>
  <td width="27%" style="text-align: center; vertical-align: middle;">
   Full Model
  </td>
  <td width="23%" style="text-align: center; vertical-align: middle;">
    <font color="green" size="5"> <b> &#10004; </b> </font>
  </td>
  <td width="26%" style="text-align: center; vertical-align: middle;">
    <font color="green" size="5"> <b> &#10004; </b> </font>
  </td>
  <td width="23%" style="text-align: center; vertical-align: middle;">
    <font color="green" size="5"> <b> &#10004; </b> </font>
  </td>
 </tr>
</table>
</figure>
<br>

## Diskretisierung 

### Grundwassermodell

> 

Reviewer: Schau dir mal das Preview an. Keine Ahnung,was da schiefgegangen ist

</figure><p>

Die Mangroven entnehmen dem Untergrund Bodenwasser aus einer Tiefe von 40 cm bis 80 cm unter der Geländeoberkante. <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_3">Abbildung 3</a> zeigt das Modellgebiet (Grau) und den Bereich der Wasserentnahme durch die Mangroven (Blau). Zu beachten ist die 50-fache Skalierung in z-Richtung. 

<figure>
<a name="Abbildung_3"></a>
<img src="/pictures/exmouth_gulf/model_area_legend.png">
<figcaption><font size = "1"><i><b>Abbildung 3:</b> Bereich der Wasserentnahme durch Mangroven</i></font></figcaption>
</figure><p>

Zeitlich diskretisiert wird das Grundwassermodell mit einer Zeitschrittlänge von einer Stunde. Der Tidenhub als dynamische Randbedingung wird mit der Zeitreihe der Jahre 1991 bis 1993, die über die gesamte Modelllaufzeit immer wieder wiederholt wird, abgebildet.

### Baumwachstumsmodell

Da jede Mangrove als einzelnes Individuum abgebildet wird findet eine räumliche Diskretisierung im eigentlichen Sinne nicht statt. Zeitlich wird das Baumwachstumsmodell mit einer Zeitschrittlänge von einem halben Jahr (1 a = 365.25 d) diskretisiert.

## Randbedingungen Grundwassermodell

> Der Salzgehalt des Meerwassers wurde mit 50 g/kg festgelegt, das Bodenwasser am landseitigem Ende des Transsekts bekam eine Salinität von 70 g/kg zugewiesen. Durch die Transpiration der Mangroven erhöhen diese lokal den Salzgehalt. Der Wasserstand wird in Form des hydrostatischen Drucks an der see- und landseitigen Kante des Modellgebiebts bestimmt. Zur Abbildung der Gezeiten wurde der seeseitige Wasserstand dabei als dynamische Randbedingung 1. Art in das Modell integriert. Als Datengrundlage dienten hier die Wasserstandsmessungen des Deparment of Transport der Government of Western Australia. Der landseitige Wasserstand wird über eine konstante Randbedingung 1. Art abgebildet. MARKER Wird die Evaporation nur in Form der landseitigen RB abgebildet? Oder gibt es zusätzlich flächige RB 2. Art oder Senken die die Verdunstung abbilden? Zufluss in Form von Niederschlag wird nur über landseitige RB mitberücksichtigt oder? MARKER Zuflüsse in Form von Niederschlag wurden nicht als eigene Randbedingung abgebildet, sondern sollen mit über die Randbedingung 1. Art am landseitigen Modellende integriert werden. Die hier nicht erwähnten Modellseiten werden alle als No-Flow-Randbedingung definiert. Eine schematische Darstellung des Modellgebiets stellt auch <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_1">Abbildung 1</a>

Reviewer: ```suggestion
Der Salzgehalt des Meerwassers wurde mit 50 g/kg festgelegt, das Bodenwasser am landseitigem Ende des Transsekts bekam eine Salinität von 70 g/kg zugewiesen. Durch die Transpiration der Mangroven erhöhen diese lokal den Salzgehalt. Der Wasserstand wird in Form des hydrostatischen Drucks an der see- und landseitigen Kante des Modellgebiebts bestimmt. Zur Abbildung der Gezeiten wurde der seeseitige Wasserstand dabei als dynamische Randbedingung 1. Art in das Modell integriert. Als Datengrundlage dienten hier die Wasserstandsmessungen des Deparment of Transport der Government of Western Australia. Der landseitige Wasserstand wird über eine konstante Randbedingung 1. Art abgebildet. Die Evapotranspiration der Bäume wird durch Senken im Bereich der Wurzeln abgebildet (Vgl Abbildung 3). Zufluss in Form von Niederschlag wird indirekt über den Salzgehalt am landseitige Rand des Modellgebiets berücksichtigt. Zuflüsse in Form von Niederschlag wurden nicht als eigene Randbedingung abgebildet, sondern sollen mit über die Randbedingung 1. Art am landseitigen Modellende integriert werden. Die hier nicht erwähnten Modellseiten werden alle als No-Flow-Randbedingung definiert. Eine schematische Darstellung des Modellgebiets stellt auch <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_1">Abbildung 1</a>
```

# Ergebnisse

> In diesem Projekt wurden mit Hilfe des pyMANGA-Modells zwei Prozesse genauer Betrachtet. Zum einen sollte die Entwicklung der typischen Strukturen in Mangrovenwäldern abgebildet werden, zum anderen sollte das Wachstumsverhalten der beiden Mangrovenarten unter verschiedenen Umweltbedingungen Untersucht werden. Im nachfolgenden werden die Ergebnisse des Projekts kurz zusammengefasst.

Reviewer: Vielleicht lieber "In dieser Arbeit..:" als "In diesem Projekt..."?

<p>

Im nachfolgenden Video wurde das Modellgebiet in zehn Sektoren unterteilt. Dargestellt wird die dynamische Entwicklung der Mangrovenpopulation und der Salzkonzentration im Bodenwasser sowie die Biomasse der Mangroven in den einzelnen Sektoren. Im Vergleich zur vorhergehenden Visualisierung ist in diesem Video eine Hauptursache der Ausbildung der typischen Waldstruktur zu erkennen, nämlich die Aufkonzentrierung des Salzgehalts im Bodenwasser in einem bestimmten Bereich. Die hohe Korrelation zwischen Salzkonzentration und Biomasse in den einzelnen Sektoren ist bereits ab einer Modelllaufzeit von 40 Jahren zu erkennen. Bereits ab 100 Jahren ist die sich ausbildende, für Mangrovenwälder typische Struktur erkennbar.

<br>

<figure>
<iframe src="https://player.vimeo.com/video/481362688" width="640" height="360" frameborder="1" allow="autoplay; fullscreen" allowfullscreen></iframe>
<figcaption><font size = "1"><i><b>Video 1:</b> Dynamische Entwicklung der Mangrovenpopulation und Salzkonzentration im Bodenwasser über die Modellierungszeit</i></font></figcaption>
</figure>

<br>

> Die Ergebnisse des "<b>Full Models</b>" stimmen mit den gemessenen Felddaten qualitativ überein (siehe <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_4">Abbildung 4</a>). Dies trifft sowohl auf das Baumhöhenprofil (siehe <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_4">Abbildung 4 A</a>) als auch auf das Profil des Salzgehalts des Porenwassers (siehe <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_4">Abbildung 4 B</a>) in dem untersuchten Transekt zu. Insbesondere die Variation des Porenwassersalzgehalts konnte gut abgebildet werden (<a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_4">Abbildung 4 A</a>). Ein Vergleich der Ergebnisse des "Full Models" mit den Ergebnissen der beiden Modellvarianten "Model Without Feedback" und "Model Without Tide" zeigt eine deutlich schlechtere Wiedergabe der gemessenen Felddaten durch die beiden einfacheren Modelle (siehe <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_4">Abbildung 4 C und 4 D</a>). 

Reviewer: In der aktuellen Paper-Version habe ich die Übereinstimmung quantifiziert. Die R2-Werte findest du im entsprechenden Kapitel der Ergebnisse.

Um die Auswirkungen der Berücksichtung der zeitlichen Dynamik der Gezeiten und der Pflanzenwasserentnahme auf den Salzgehalt im Porenwasser zu untersuchen, wurde diese mit folgender Formel normiert:

<figure>
<div align="center">
<img src="/pictures/exmouth_gulf/formel.png" width="50%">
</div>
</figure><p>

Diese relativen Auswirkungen sind in nachfolgender <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_5">Abbildung 5</a> für die Baumhöhe und den Porenwassersalzgehalt abgebildet. Ein Wert von Null würde bedeuten, dass sich die Ergebnisse zwischen Full Model und dem jeweiligen vereinfachtem Modelltyp nicht unterscheiden. Je größer der Wert wird, umso höher ist die Abweichung.

<figure>
<a name="Abbildung_5"></a>
<img src="/pictures/exmouth_gulf/Figure_3_2.png">
<figcaption><font size = "1"><i><b>Abbildung 5:</b> Relative Auswirkung der Nichtberücksichtigung des Tidenhubs ("Model Wihtout Tide") und der Pflanzenwasserentnahme ("Model Without Feedback")</i></font></figcaption>
</figure><p>

> Aufgrund der größeren Auswirkungen des Tidenhubs im seenahen Bereich kann das Modell "<b>Without Tide</b>" sowohl die Bäumhöhen als auch den Porenwassersalzgehalt hier nur mit reltiv großer Abweichung im Vergleich zum "<b>Full Model</b>" abbilden. Je weiter man sich aber in Richtung Festland bewergt, desto geringer werden die Wasserstandsschwankungen aufgrund der Gezeiten. Die Baumhöhen und Salzgehalte können in diesem Bereich (x > 75 m) mit geringeren relativen Abweichungen zum Full Model abgebildet werden.

Reviewer: ```suggestion
Aufgrund der größeren Auswirkungen des Tidenhubs im seenahen Bereich kann das Modell "<b>Without Tide</b>" sowohl die Bäumhöhen als auch den Porenwassersalzgehalt hier nur mit relativ großer Abweichung im Vergleich zum "<b>Full Model</b>" abbilden. Je weiter man sich aber in Richtung Festland bewergt, desto geringer werden die Wasserstandsschwankungen aufgrund der Gezeiten. Die Baumhöhen und Salzgehalte können in diesem Bereich (x > 75 m) mit geringeren relativen Abweichungen zum Full Model abgebildet werden.
```

Das Modell "<b>Without Feedback</b>" hat insbesondere im mittleren bis landseitigem Bereich (60 m < x < 165 m) des Transsekts Probleme, die Wachstumshöhe der Mangroven so abzubilden, wie es das "<b>Full Model</b>" macht. 

## Speziendominanz

> Im vorangegangenem Abschnitt wurde gezeigt, dass pyMANGA mit der Berücksichtigung von Salzkonzentration im Bodenwasser und des Tidenhubs in der Lage ist, die für Mangrovenwälder typischen Waldstrukturen abzubilden. Mit Hilfe der umfangreichen Parametrisierungsmöglichkeiten des Baumwachstumsmodells (siehe hierzu auch <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Parametrisierung">diesen Abschnitt</a>) kann mit pyMANGA auch das Wachstum einzelner bestimmter Individuenarten untersucht werden. Verschiedene Mangrovenarten zum Beispiel weisen verschiedene Toleranzen gegenüber zu hohen Salzgehalten auf. In diesem Projekt wurde das Wachstumsverhalten der beiden Arten Avicennia marina (<a href="https://www.biologie-seite.de/Biologie/Avicennia_marina">"graue Mangrove"</a>) und Rhizophora mangle (<a href="https://www.biologie-seite.de/Biologie/Rote_Mangrove">"rote Mangrove"</a>) genauer betrachtet.

Reviewer: ```suggestion
Im vorangegangenem Abschnitt wurde gezeigt, dass pyMANGA mit der Berücksichtigung von Salzkonzentration im Bodenwasser und des Tidenhubs in der Lage ist, die für Mangrovenwälder typischen Waldstrukturen abzubilden. Mit Hilfe der umfangreichen Parametrisierungsmöglichkeiten des Baumwachstumsmodells (siehe hierzu auch den Abschnitt <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Parametrisierung">Parametrisierung</a>) kann mit pyMANGA auch das Wachstum einzelner bestimmter Individuenarten untersucht werden. Verschiedene Mangrovenarten zum Beispiel weisen verschiedene Toleranzen gegenüber zu hohen Salzgehalten auf. In diesem Projekt wurde das Wachstumsverhalten der beiden Arten Avicennia marina (<a href="https://www.biologie-seite.de/Biologie/Avicennia_marina">"graue Mangrove"</a>) und Rhizophora mangle (<a href="https://www.biologie-seite.de/Biologie/Rote_Mangrove">"rote Mangrove"</a>) genauer betrachtet.
```

<a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Abbildung_6">Abbildung 6</a> zeigt die Speziendominanz dieser beiden Mangrovenarten bei unterschiedlichen Salzkonzentration (siehe <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Tabelle_6">Tabelle 6</a>) im Bodenwasser. Die verschiedenen in der Abbildung dargestellten Setups unterscheiden sich nur hinsichtlich der Randbedingungen der seeseitigen und landseitigen Salzkonzentrationen des Bodenwassers. Für die Betrachtung der Speziendominanz im Modellgebiet wir die Speziendominanz d eingeführt und wie folgt definiert:

<figure style="width:75%">
<div align="center">

> <img src="/pictures/exmouth_gulf/formel_normierung_speziendominanz.jpg" style="width:45%">

Reviewer: Dann hier entweder ohne summe, oder genau wie die nachfolgende formel...

Mit Hilfe der Sensitivtätsanalyse des Modells hinsichtlich der Speziendominanz konnte gezeigt werden, dass die Artenzusammensetzung durch die gekoppelte Betrachtung von Bodenwasserhaushalt und Pflanzenwasserentnahme über die Systemgrenze hinweg beschrieben werden kann. Durch die Variation von nur zwei Parametern (vgl. <a href="/de/docs/beispielmodell_exmouth_gulf/ergebnisse/#Tabelle_3">Tabelle 3</a>), die sich direkt auf die Wasseraufnahme der Bäume auswirken, konnten typische Zonierungsmuster in Mangroven-Mischwäldern reproduziert werden. Dies gelang auch mit nur grob geschätzten pflanzenspezifischer Parameter für eine der beiden Arten. Werden die Randbedingungen der Salzkonzentrationen an land- und seeseitiger Seite beide sehr hoch oder sehr niedrig gewählt bilden sich überwiegend Wälder aus Monokulturen aus. Wählt man einen moderaten Mittelwert der Salinitäten bilden sich Mischwälder aus beiden Arten. Diese weisen Zonen mit klarer Dominanz einer Art auf, die durch scharfe Übergänge getrennt sind. Diese Übergänge hängen dabei nachweislich vom Bodendwassersalzgehalt ab. Die Spezienzusammensetzung im Modell stimmt mit den gemessenen Felddaten überein.

#Ausblick

> In der Literatur gibt es Hinweise darauf, dass sich Mangroven über die Zeit an ihre Umweltbedingungen anpassen. Eine niedrigere Salzkonzentration im Bodenwasser sorgt für eine höhere Xylemleitfähigkeit und damit einer größeren Transpiration. Gleichzeitig sorgt eine geringe Salzkonzentration im Untergrund aber auch für höhere Blattwasserpotential, die die Transpiration hemmen. Diese sich gegenseitig ausgleichenden Prozesse sorgen für insgesamt etwa gleichbleibende Transpirationsraten bei unterschiedlichen Salinitäten des Porenwassers. Eine genauere Untersuchung dieser Prozesse kann dabei helfen, MARKER nicht sicher, was physiological plasticity genau ist. Die Fähigkeit des Baumes Prozesse an die Umwelt anzupassen? MARKER

Reviewer: physiological plasticity = Die Möglichkeit die physiologie des baumes an umweltbedingungen anzupassen.

</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline"><em>k</em><sub><em>m</em></sub></span></td>

> <td style="text-align: left;">Maintenance cost per biomass MARKER: Übersetzung</td>

Reviewer: Resourcenbedarf zur Biomasseerhaltung

Author: Done
